storage/database/db_migration.go

```diff
@@ -75,29 +57,113 @@ loop:
 			return errors.Wrap(err, "failed to put batch")
 		}
 
+		if dstBatch.ValueSize() > IdealBatchSize {
+			if err := dstBatch.Write(); err != nil {
+				return err
+			}
+			dstBatch.Reset()
+		}
+
 		// make a report
-		if fetched%reportCycle == 0 {
+		if fetched%(IdealBatchSize*20) == 0 {
```

Comment:
Why did you change `reportCycle` to `IdealBatchSize*20`?

Reply:
The log are too much. So I changed it.

Reply:
If so, how about increasing `reportCycle` as it is not used now?

Reply:
Reflected.